test/acsengine.deploy

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env groovy
+
+node {
+  withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'AZURE_CLI_SPN_ACS_TEST',
+                  passwordVariable: 'SPN_PASSWORD', usernameVariable: 'SPN_USER']]) {
+    timestamps {
+      wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm']) {
+        env.GOPATH="${WORKSPACE}"
+        def clone_dir = "${env.GOPATH}/src/github.com/Azure/acs-engine"
+        env.HOME=clone_dir
+        env.ORCHESTRATOR = "${ORCHESTRATOR}"
+        String locations_str = "${LOCATIONS}"
+        
+        if(locations_str.equals("all")) {
+          locations_str = "\
+australiaeast australiasoutheast \
+brazilsouth \
+canadacentral canadaeast \
+centralindia southindia westindia\
+centralus eastus2 eastus northcentralus southcentralus westcentralus westus2 westus \
+eastasia southeastasia \
+japaneast japanwest \
+northeurope westeurope \
+uksouth ukwest"
```

Comment:
Make to run tests against at least all these regions. Technically these are ACS regions but its good run ACS Engine tests in these regions. Especially US and Europe regions as they tend to get a lot more traffic.

australiasoutheast
northeurope       
brazilsouth       
australiaeast     
japaneast         
northcentralus    
westus            
eastasia          
eastus2           
southcentralus    
southeastasia     
eastus            
westeurope        
Centralus
japanwest        